metastore/src/main/scala/com/gu/tableversions/metastore/VersionPaths.scala

```diff
@@ -31,8 +31,9 @@ object VersionPaths {
       partitionPath
     else {
       def normalised(path: String): String = if (path.endsWith("/")) path else path + "/"
+
       def versioned(path: String): String = s"$path${version.label}"
+
```

Comment:
I don't disagree with the change, but this one is actually causing a conflict with the master branch, so it might just be easier to leave this as it is for now...